stellarphot/settings/models.py

```diff
@@ -543,7 +548,7 @@ class Observatory(BaseModelWithTableRep):
         WithPhysicalType("length"),
         Field(
             description="Elevation of the observatory",
-            examples=["1000 m", "1 km", "3.241e-14 pc"],
+            examples=["1000 m", "1 km", "3.241e-14 pc", "1e12 nm"],
```

Comment:
Really, you added a trillion nm?  Was this for the spectroscopists in the audience?  You might as well give the measurement as 1e5 cm to make the stellar astrophysicists happy. ;)

Reply:
I'm happy to change it to 1e5 cm...I just wanted a 4th option so I could test it